Makefile

```diff
@@ -1,21 +1,56 @@
 SEVERITIES = HIGH,CRITICAL
 
+ifeq ($(ARCH),)
+ARCH=$(shell go env GOARCH)
+endif
+
+ifeq ($(KUBERNETES_VERSION),)
+KUBERNETES_VERSION=v1.21.1
+endif
+
+BUILD_META=-build$(shell date +%Y%m%d)
+ORG ?= rancher
+PKG ?= github.com/kubernetes/kubernetes
+SRC ?= github.com/kubernetes/kubernetes
+TAG ?= v1.21.1$(BUILD_META)
+
+ifneq ($(DRONE_TAG),)
+TAG := $(DRONE_TAG)
+endif
+
+ifeq (,$(filter %$(BUILD_META),$(TAG)))
+$(error TAG needs to end with build metadata: $(BUILD_META))
+endif
+
+.PHONY: image-build
+image-build:
+	docker build \
+		--pull \
+		--build-arg ARCH=$(ARCH) \
+		--build-arg PKG=$(PKG) \
+		--build-arg SRC=$(SRC) \
+		--build-arg TAG=$(TAG:$(BUILD_META)=) \
+		--tag $(ORG)/hardened-kubernetes:$(TAG)-linux-$(ARCH) \
+	.
+
 .PHONY: all
 all:
 	docker build \
 	--build-arg K8S_TAG=$(shell echo $(TAG) | grep -oP "^v(([0-9]+)\.([0-9]+)\.([0-9]+))") \
-	--build-arg TAG=$(TAG) -t rancher/hardened-kubernetes:$(shell echo $(TAG) | sed -e 's/+/-/g') .
+	--build-arg TAG=$(TAG) -t $(ORG)/hardened-kubernetes:$(shell echo $(TAG) | sed -e 's/+/-/g') .
 
 .PHONY: image-push
 image-push:
-	docker push rancher/hardened-kubernetes:$(TAG) >> /dev/null
+	docker push $(ORG)/hardened-kubernetes:$(TAG)-linux-$(ARCH) >> /dev/null
 
 .PHONY: scan
 image-scan:
-	trivy --severity $(SEVERITIES) --no-progress --skip-update --ignore-unfixed rancher/hardened-kubernetes:$(TAG)
+	trivy --severity $(SEVERITIES) --no-progress --skip-update --ignore-unfixed $(ORG)/hardened-kubernetes:$(TAG)
 
 .PHONY: image-manifest
 image-manifest:
-	docker image inspect rancher/hardened-kubernetes:$(TAG)
-	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create rancher/hardened-kubernetes:$(TAG) \
-		$(shell docker image inspect rancher/hardened-kubernetes:$(TAG) | jq -r '.[] | .RepoDigests[0]')
+	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest create --amend \
+		$(ORG)/hardened-kube-proxy:$(TAG) \
+		$(ORG)/hardened-kube-proxy:$(TAG)-$(ARCH)
+	DOCKER_CLI_EXPERIMENTAL=enabled docker manifest push \
+		$(ORG)/hardened-kube-proxy:$(TAG)
```

Comment:
New line

Reply:
good catch, i didnt even use this cause i used the manifest plugin in drone but ill fix it for people to test with is